design-system/gulpfile.js

```diff
@@ -125,18 +99,7 @@ function css() {
   return gulp
     .src(src_paths.css)
     .pipe(sourcemaps.init({ loadMaps: true }))
-    .pipe(cssimport(importOptions))
-    .pipe(
-      postcss(
-        [
-          postcssCustomMedia(),
-          postcssCustomProperties(),
-          postcssNesting(),
-          postcssCalc()
-        ]
-      )
-    )
-    .pipe(autoprefixer())
+    .pipe(postcss())
```

Comment:
If we leave the config out, it'll magically find **postcss.config.js** instead 🎉